packages/hardhat-vyper/src/compilation.ts

```diff
@@ -73,8 +73,7 @@ export async function compile(
       // TODO this might not work on windows, maybe we should use `slash` here
```

Comment:
We should probably export the `source-names.ts` too, right?

Reply:
This variable doesn't seem to be used anyway.

Reply:
Oh, but it should be used 😅